packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -329,6 +352,14 @@ export class Stream extends StreamBase {
 
     throw new Error(`Unexpected 'encryptionType': ${encryptionType}`);
   }
+
+  private getEncryptionCondition(conditionName: string): IResolvable {
```

Comment:
I would inline this function, I don't see much gain from it. If it means changing to a multi-line return in `parseEncryption` like this:

```ts
return {
  streamEncryption: <lots of stuff>,
  encryptionKey: undefined,
}
```

I'm fine with it (BTW, I'm pretty sure `encryptionKey: undefined` is redundant - you can leave it out, and it will have the same effect).

Reply:
done